commands/release.js

```diff
@@ -64,7 +64,7 @@ export async function command() {
 	});
 
 	const newVersionIsLargestVersion = stableVersions.every(version => {
-		return gt(newVersion, version);
+		return gt(prereleaseComponents, version);
```

Comment:
Does this work?
`gt` docs state it takes a single version string as the first argument, `prereleaseComponents` is an array.

Reply:
no I sneakily closed this pr 🙈